nntrainer/layers/layer_context.cpp

```diff
@@ -694,6 +695,10 @@ std::string RunLayerContext::getKernelName(LayerKernel layerKernel) {
     return "addition_cl";
   case LayerKernel::ADD_FP16:
     return "addition_cl_fp16";
+  case LayerKernel::SWIGLU:
```

Comment:
Not this PR, but we need to rethink that this Kernel Name and actual kernel object placed in RunContext. Originally RunContext is just for the input / output / weight and Tensors to compute.. and I think this ClKernel should be handled at cl layer itself as in CPU Kernel.